staging/src/k8s.io/kubelet/config/v1beta1/types.go

```diff
@@ -482,6 +482,12 @@ type KubeletConfiguration struct {
 	// Default: true
 	// +optional
 	SerializeImagePulls *bool `json:"serializeImagePulls,omitempty"`
+	// MaxParallelImagePulls sets the maximum number of image pulls in parallel.
+	// This field is ignored if SerializeImagePulls is true.
+	// Setting it to 0 means no limit.
+	// Default: 0
+	// +optional
+	MaxParallelImagePulls int32 `json:"maxParallelImagePulls,omitempty"`
```

Comment:
I'm surprised that NO similar fields like `MaxPods` fields are being validated to be non-negative numbers. Specifically for `MaxParallelImagePulls` the code will treat negative as `0` as written, so it is not a problem with this PR. Just a general comment

Reply:
If we want to use 0 as no limit and in the future we use N as default, we should using `int32*` here. Then this can be changed to 0.

If the defaulting is like below using `int32` type, the 0 will be changed to N by default. 0 will not stand for no limit then.
```
if xx.MaxParallelImagePulls == 0 {
	xx.MaxParallelImagePulls = N
}
```

So this should be  `int32*`
```
if xx.MaxParallelImagePulls == nil {
	xx.MaxParallelImagePulls = utilpointer.Int32Ptr(N)
}
```

Reply:
uh good catch! Will update the code

Reply:
Updated the code. PTAL!